src/main/java/com/epam/lab/payments/model/IndexController.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.lab.payments.model;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Controller
+public class IndexController {
+
+    private final CardRepository cardRepository;
+
+    public IndexController(CardRepository cardRepository) {
+        this.cardRepository = cardRepository;
+    }
+
+    @GetMapping("/")
+    public ModelAndView index() {
+        Map<String, String> model = new HashMap<>();
```

Comment:
I don't understand, why it's here?

Reply:
This is temporal DB stub, we delete it later. Real DB is already created and will replace all this junk very soon